### PR TITLE
Adds render command

### PR DESCRIPTION
**Pull request by toopy**

### CODE REVIEW

**Comment by toopy** [comment]
@bersace @xcambar .. OPM ?

File: bag8/cli.py [hunk 1]
```diff
@@ -175,6 +177,15 @@ def push(project):
 
 @bag8.command()
 @click.argument('project', default=cwdname)
+@click.argument('output', type=click.File('wb'), default='fig.yml')
+def render(output, project):
+    """Renders fig.yml like content to out file, default: fig.yml.
+    """
+    json.dump(Yaml(Project(project)).data, fp=output, indent=2, sort_keys=True)
```
Comment (xcambar) [inline bag8/cli.py]: `json.dump`? You sure?

**Comment by xcambar** [comment]
yep

**Comment by bersace** [comment]
OPM

**Pull request merged**